mimic-navigation/navigation.md

```diff
@@ -0,0 +1,97 @@
+# Navigating Mimic
+
+Mimic is a single-window application. The interface is divided into four main parts:
+
+- **Action Bar**
+- **Routes**
+- **Logs**
+- **Status bar**
+
+Every part of the application was designed to give you enough space for showing information.
+
+## Action Bar
+
+![alt text](../Images/actionbar.png 'Action bar')
+
+On the Action Bar, the following options are available:
+
+- **Projects** Button - Create a new project, switch between projects.
+
+- **Edit** Icon - Edit the project name.
+
+- **Start Server** Button - Start the server, stop the server.
+
+- **Plus(+)** Button - Add a route to the project.
+
+- **Menu** Button - Projects settings, delete all routes, delete project.
+
+## Routes
+
+![alt text](../Images/routes.png 'Routes')
```

Comment:
[int] _alt text_ 👀

There are a few more of those, you can just `cmd + f` to catch'em all!